website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,122 @@
+<link rel="stylesheet" href='/static/css/pages/remove-contributor-page.css'>
+<div id="removeContributor" class="modal fade">
+    <div data-bind="css: modalSize">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+            <div class="modal-body" >
+                <div data-bind="if: canRemoveNode() && !pageChanged()">
+                    <!-- remove page -->
+                    <div data-bind='if:page() === REMOVE'>
+                        <div class="form-group">
+                            <span>Do you want to remove <b>{{contributorToRemove()["fullname"]}}</b> from <b>{{title}}</b>, or from <b>{{title}}</b> and every component in it.</span>
+                        </div>
+                        <div id="remove-page-radio-buttons" class="col-md-8" align="left">
+                            <div class="radio">
+                                <label><input type="radio" name="radioBoxGroup" data-bind="checked:deleteAll, checkedValue: false" checked>Remove  <span class="f-w-lg" >{{contributorToRemove()["fullname"]}}</span> from  <span class="f-w-lg" data-bind="text: title"></span></label>
+                            </div>
+
+                            <div class="radio">
+                                <label><input  type="radio" name="radioBoxGroup" data-bind="checked: deleteAll, checkedValue: true" >Remove <span class="f-w-lg" >{{contributorToRemove()["fullname"]}}</span> from <span class="f-w-lg" data-bind="text: title"></span> and every component in it.</label>
+                            </div>
+                        </div>
+
+                    </div><!-- end remove page -->
+                    <!-- removeNoChildren page -->
+                    <div data-bind='if:page() === REMOVE_NO_CHILDREN'>
+                        <div class="form-group" data-bind="if:contributorToRemove">
+                            <span>Remove <b>{{contributorToRemove()["fullname"]}}</b> from {{title}}"?</span>
```

Comment:
There's an extra quotation mark right before the question mark.

![screen shot 2016-01-12 at 3 05 09 pm](https://cloud.githubusercontent.com/assets/134455/12275420/0f217080-b93e-11e5-82b9-49a314541467.png)